plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/ideMaven/MavenRunnerUtils.kt

```diff
@@ -117,7 +117,7 @@ private fun runMavenCopyDependencies(
         } catch (t: Throwable) {
             val error = "Maven Copy: Unexpected error when executing bundled Maven copy dependencies"
             copyTransformRunnable.setExitCode(Integer.MIN_VALUE) // to stop looking for the exitCode
-            logger.error(t) { error }
+            logger.info(t) { error }
```

Comment:
This as an error was apparently confusing people, since we don't stop the transformation when copy-deps fails we were advised to make this an info message instead.

Reply:
I suppose that would work to, doesn't really matter as long as it's not an `error`